File: hackerrank/javascript/week1/minimaxsum.js
function miniMaxSum(arr) {
    // Write your code here
    let min = Math.min(...arr)
    let max = Math.max(...arr)
    let minTotal = 0
    let maxTotal = 0
    for (let i = 0; i < arr.length; i++) {
        if (min === max) {
            maxTotal = max * 4
            minTotal = min * 4
        }
        if (arr[i] !== min) {
            maxTotal += arr[i]
        }
        if (arr[i] !== max) {
            minTotal += arr[i]
        }
    }
    console.log(minTotal, maxTotal)
}

File: hackerrank/python/week1/minimaxsum.py
def miniMaxSum(arr):
    # Write your code here
    arr.sort()
    minSum = 0
    maxSum = 0
    for i in range(0, len(arr) - 1):
        minSum += arr[i]
    for i in range(1, len(arr)):
        maxSum += arr[i]
    print(minSum, maxSum)
